engine/src/main/java/org/entando/entando/aps/system/services/page/PageTokenManager.java

```diff
@@ -108,15 +110,6 @@ public String decrypt(String property) {
 		return null;
 	}
 
-	public void updateHash() throws Exception {
-		try {
-			String param = this.generateRandomHash();
-			this.generateInternalSaltAndPassword(param);
-		} catch (Throwable t) {
-			throw new Exception("error in updateHash", t);
-		}
-	}
```

Comment:
did we check if we had any other class actually calling this public method before deleting it?